app.js

```diff
@@ -24,7 +24,7 @@ app.use((req, res, next) => {
     next();
 });
 
-app.use('/assets', express.static('assets'));
+app.use(express.static('static'));
```

Comment:
update the `help` embedded message logo url. and delete the old assets folder

Reply:
Updated